theme/_layouts/post.html

```diff
@@ -2,6 +2,7 @@
 layout: default
 ---
 
+{% include sidebar.html class="post" %}
```

Comment:
It looks like with the `post` class this breaks on non-mobile layouts, showing the sidebar over everything all the time. A quick local test seems to indicate that using `blog` (the same that's used for the blog index page) works correctly.

```suggestion
{% include sidebar.html class="blog" %}
```

Reply:
Cool, I originally had `blog` but wasn't sure if that was incorrect. Thanks for the quick review!